sdks/python/apache_beam/io/filebasedsource.py

```diff
@@ -91,18 +92,45 @@ def default_output_coder(self):
 class FileBasedSource(iobase.BoundedSource):
   """A ``BoundedSource`` for reading a file glob of a given type."""
 
-  def __init__(self, file_pattern, min_bundle_size=0, splittable=True):
+  def __init__(self,
+               file_pattern,
+               min_bundle_size=0,
+               # TODO(slaven): Add support for AUTO once cl/130858239 is in.
```

Comment:
# TODO(slaven): Add support for automatically determining the type of compressed files.